tests/test_sidecar.py

```
@@ -49,9 +49,10 @@ def test_zero_coordinates(tmp_path: Path) -> None:
     json_path = tmp_path / "a.jpg.json"
     json_path.write_text(json.dumps(sample), encoding="utf-8")
     meta = parse_sidecar(json_path)
-    assert meta.latitude == 0.0
-    assert meta.longitude == 0.0
-    assert meta.altitude == 10.0
+    # 0/0 coordinates should be filtered out as unreliable
+    assert meta.latitude is None
```

Ruff (0.12.2)

Use of `assert` detected

(S101)